src/main/java/com/example/demo/controller/ItemController.java

```diff
@@ -22,12 +25,18 @@
 @RequestMapping("/items")
 public class ItemController {
 
+	private final ItemService itemService;
+
 	@Autowired
-	ItemService itemService;
+	public ItemController(ItemService itemService) {
+		this.itemService = itemService;
+	}
 
 	/* Top */
 	@RequestMapping("/index")
```

Comment:
HTTPメソッドGETだと思うのでGetMappingにしましょう。
```suggestion
	@GetMapping("/index")
```